lunes_cms/cms/utils.py

```diff
@@ -61,11 +61,20 @@ def document_to_string(doc):
     :rtype: str
     """
     alt_words = [str(elem) for elem in doc.alternatives.all()]
+    has_foto = "\U0001F4F7" if doc.document_image.all() else ""
```

Comment:
```suggestion
    has_foto = " " + "\U0001F4F7" if doc.document_image.all() else ""
```
I think the icon needs a bit more breathing-room to the left